package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Node.js package for Innovation Sandbox",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && mocha --timeout 30000",
+    "test": "npm run lint && mocha test/**/*.js --timeout 30000",
```

Comment:
mocha reads test/ directory automatically, change not required

Reply:
Used recursive flag instead.